hyperlane.Dockerfile

```
@@ -0,0 +1,67 @@
+  rm -rf /var/lib/apt/lists/* && \
+  npm uninstall -g yarn && \
+  npm cache clear --force && \
+  cd - && \
+  rm -rf hyperlane-monorepo
+
+WORKDIR /app
+
+# anvil
+COPY --from=rust-builder /root/.foundry/bin/* /usr/bin
+# rust and validators
+COPY --from=rust-builder /usr/bin/relayer /usr/bin/validator /usr/bin
```

Hadolint (2.12.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)